src/components/statistics/Statistics.tsx

```diff
@@ -22,17 +24,30 @@ function Statistics() {
   Api.makeIncidentRequest(UrlConfig.urlIncident).then(result => setSevenDayIncident(result));
 
   const colorOfElement = infectionsDelta > 10000 ? "#c42929" : "black";
-  let information = "Derzeit werden die Infektionszahlen anhand des 7-Tage-Inzidenz-Wertes eingefärbt.\nAlles über 100 wird rot eingefärbt.\nAlles zwischen 50 und 100 wird gelb eingefärbt.\nAlles unter 50 wird grün eingefärbt.\n\nDerzeitiger Wert: "+sevenDayIncident;
-  
+  let status = "Kritisch";
   let colorOfInfections = "black";
+  let currentDate = new Date(Date.now());
+
+  /*
+      The user visits on the 30.01 @ 1:00am
+      The website would show that the current data is from the 30.01 @ 3:00am.
+      This should be able to fix this problem
+  */
+  if(currentDate.getHours() < 3){
+      currentDate.setDate(currentDate.getDate() - 1);
+  }
+
   if(sevenDayIncident > 100){
```

Comment:
Änder das so, dann sparst du dir ein else-statement:
```typescript
colorOfInfections = "#e0d916";
status = "Gefährlich";

if (sevenDayIncident > 100) {
    colorOfInfections =  "#c42929";
    status = "Kritisch";
} else if (sevenDayIncident < 50) {
    colorOfInfections =  "#43c429";
    status = "Ungefährlich";
}
```

Denk auch dran, dass man in Typescript Strichpunkte benutzt
Vielleicht switch-case? Weiß aber nicht, ob das kompakter ist

Reply:
Done